### PR TITLE
SUIT-12173 Fix --inspect --inspectBrk in js api when used without port number

### PR DESCRIPTION
**Pull request by AntonSBilan**
Fix --inspect --inspectBrk in js api when used without port number

### CODE REVIEW

File: lib/utils/testLauncherHelper.js [hunk 1]
```diff
@@ -145,12 +145,26 @@ function getDeviceLogStream(device, userArgs, startTime, logDir) {
...
 /**
- * Verify the debug port parameter, can be port number, boolean or undefined
+ * Check if debug argument presents and return debug option
  * @param {Object} ownArgw - implicitly derived launched parameters
- * @returns {number|undefined|boolean}
+ * @returns {string | boolean}
  */
-function getDebugPort(ownArgw) {
-	return ownArgw.inspectBrk || ownArgw.inspect;
+function getDebugOption({inspectBrk, inspect}) {
```
Comment (givehug) [inline lib/utils/testLauncherHelper.js]: Please add tests for these helpers and for parseOptions

File: lib/utils/testLauncherHelper.js [hunk 2]
```diff
@@ -145,12 +145,26 @@ function getDeviceLogStream(device, userArgs, startTime, logDir) {
...
+		'--inspect-brk': inspectBrk,
+		'--inspect': inspect,
+	}));
+
+	return debuggerType[0]
+		? `${debuggerType[0][0]}${ramda.is(Number, debuggerType[0][1]) ? '=' + debuggerType[0][1] : ''}`
+		: false;
+}
+
+function isDebugMode({inspectBrk, inspect}) {
```
Comment (givehug) [inline lib/utils/testLauncherHelper.js]: Would be cleaner if isDebugMode returned boolean and then you don't have to do !!isDebugMode

**Comment by andriichumak** [comment]
@AntonSBilan , please resolve the conflicts

**Comment by AntonSBilan** [comment]
> @AntonSBilan , please resolve the conflicts

done

**Pull request merged**